.expeditor/verify.pipeline.yml

```diff
@@ -1,44 +1,83 @@
 steps:
 
-- label: run-lint-and-specs-ruby-2.4
+# Allow ruby-2.4 to softfail since it was configured so in travis too
+- label: ":ruby: 2.4"
   command:
-    - bundle install --jobs=7 --retry=3 --without docs development
+    - ruby --version
+    - bundle --version
+    - gem update --system
+    - bundle install --without development_extras development --jobs 3 --retry 3 --path vendor/bundle
+    - gem install yajl-ruby json psych
     - bundle exec rake
+  soft_fail:
+      - exit_status: 1
   expeditor:
     executor:
       docker:
-        image: ruby:2.4-stretch
-        privileged: true
+        image: ruby:2.4
```

Comment:
what OS is used in the non-debian images?

Reply:
it is running `Debian GNU/Linux 10 (buster)`

Reply:
cool